pkg/broker/server/sse.go

```diff
@@ -140,6 +162,9 @@ func (b *Broker) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 func (b *Broker) run() {
 	for {
 		select {
+		case <-b.finish:
+			log.Infoln("Broker finished")
+			return
 
 		case subscription := <-b.newClients:
```

Comment:
I have tested this PR with the code below.
```
func main() {
        client, _ := rpc.NewClient("/root/.infrakit/plugins/event")
        topics, _ := client.Topics()
        fmt.Printf("%v \n", topics)
        sub, _ := client.(spi.Subscriber)
        typed, err := sub.SubscribeOn(spi.NewTopic("timer/sec/1"))
        if err != nil {
                fmt.Errorf("%v \n", err)
        }
        for {
                select {
                case ret := <-typed:
                        fmt.Printf("%v \n", ret)
                }
        }
}
```
Then I found a problem that only one client can subscribe in one topic.
When a new client starts subscribing to a topic that another client already subscribes to, the old client will not receive any messages.
The cause of this problem is thought to be because Broker uses radix's insert, so if you try to add a new client with the same topic, old client will be overwritten.

Is this a specification?
If so, the old client's session should be closed. If not, we need to fix.
Perhaps it would be better to discuss this separately from this PR. In that case open a new issue and consider a patch.
What do you think?